src/main/java/seedu/address/model/person/Name.java

```diff
@@ -14,10 +14,9 @@
             "Person names should only contain alphanumeric characters and spaces, and it should not be blank";
 
     /*
-     * The first character of the address must not be a whitespace,
-     * otherwise " " (a blank string) becomes a valid input.
+     * Matches unicode words separated with one space, e.g. "Charles Nguyễn 你好"
      */
-    public static final String NAME_VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String NAME_VALIDATION_REGEX = "\\p{L}+( \\p{L}+)*";
```

Comment:
Is there any significance for the space in the regex?

Reply:
Because the names need to be separated with a single space